src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -82,10 +83,10 @@ static Widget _inventionListDragWidgets[] = {
     static void ResearchRidesSetup()
     {
         // Reset all objects to not required
-        for (auto objectType : TransientObjectTypes)
+        for (auto objectType : getTransientObjectTypes())
         {
-            auto maxObjects = object_entry_group_counts[EnumValue(objectType)];
-            for (int32_t i = 0; i < maxObjects; i++)
+            auto maxObjects = getObjectEntryGroupCount(objectType);
+            for (auto i = 0U; i < maxObjects; i++)
```

Comment:
This is a nitpick, but we always use a lower case `u` here:

```suggestion
            for (auto i = 0u; i < maxObjects; i++)
```

Reply:
Oh, that's surprising, considering we do use the uppercase `L`, right? I can amend if needed, though.

Reply:
Let’s briefly discuss this in the team chat.

Reply:
Ok, so I’d like to stick with the existing convention of a lower case `u`. I’ll try and add a clang-tidy rule for it. Could you update it?

Reply:
I already have :-) There's a commit changing the U's to u's already

Reply:
This one still shows up as `U` for me, maybe it was incomplete?

Reply:
Looks like the comment/thread is tied to an outdated version of the line. It has since been updated.

Reply:
Yes - by me.